src/osd/OSD.cc

```diff
@@ -9923,35 +10028,34 @@ void OSD::ShardedOpWQ::_process(uint32_t thread_index, heartbeat_handle_d *hb)
 
   // [lookup +] lock pg (if we have it)
   if (!pg) {
-    pg = osd->_lookup_lock_pg(item.first);
+    pg = osd->_lookup_lock_pg(token);
   } else {
     pg->lock();
   }
 
```

Comment:
the `pg` is still being referenced by `_process()` at multiple places. and i don't have a clear plan on how to abstract it so far. probably we should remove the pg from `pg_slot` using an abstract interface.